integration/deployment_test.sh

```diff
@@ -26,9 +26,9 @@ run_test() {
     try ${PEER_CMD} lifecycle chaincode package --lang fpc-c --label auction_test --path ${FPC_TOP_DIR}/examples/auction/_build/lib/ ${PKG}
     try ${PEER_CMD} lifecycle chaincode install ${PKG}
     PKG_ID=$(${PEER_CMD} lifecycle chaincode queryinstalled | awk "/Package ID: auction_test/{print}" | sed -n 's/^Package ID: //; s/, Label:.*$//;p')
-    try ${PEER_CMD} lifecycle chaincode approveformyorg -C ${CHAN_ID} --package-id ${PKG_ID} --name auction_test --version ${CC_VERS}
-    try ${PEER_CMD} lifecycle chaincode checkcommitreadiness -C ${CHAN_ID} --name auction_test --version ${CC_VERS}
-    try ${PEER_CMD} lifecycle chaincode commit -C ${CHAN_ID} --name auction_test --version ${CC_VERS}
+    try ${PEER_CMD} lifecycle chaincode approveformyorg -C ${CHAN_ID} --package-id ${PKG_ID} --name auction_test --version ${CC_VERS} -V ecc-vscc
```

Comment:
Wouldn't it be better to add the validation plugin inside the peer cli?  Mid/long-term some of these plugins will change for FPC, so it would be cleaner/easier from a functional interface point of view as the interface is simpler and would be the same even if we would move to MSP/X509 style validation.  As we anyway do not allow any user-provided plugins, the peer cli of course should also barf if any plugins provided (which it currently doesn't, e.g., if you provide an endorsement plugin, which by itself is also a bit of a problem?)

Reply:
> Wouldn't it be better to add the validation plugin inside the peer cli? Mid/long-term some of these plugins will change for FPC, so it would be cleaner/easier from a functional interface point of view as the interface is simpler and would be the same even if we would move to MSP/X509 style validation. As we anyway do not allow any user-provided plugins, the peer cli of course should also barf if any plugins provided (which it currently doesn't, e.g., if you provide an endorsement plugin, which by itself is also a bit of a problem?)

Created a new issue for that #372